src/screenControl.py

```diff
@@ -244,7 +247,7 @@ def getScreenDimensions(self):
 
     def getChromeBoundaries(self):
         (maxy, maxx) = self.stdscr.getmaxyx()
-        minx = CHROME_MIN_X if self.scrollBar.getIsActivated() else 0
+        minx = CHROME_MIN_X if self.scrollBar.getIsActivated() or self.mode == X_MODE else 0
```

Comment:
That would be mixing purposes.  Just because X_MODE is on doesn't mean the scroll bar is too.  I just want to indent in both cases.